src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Utils.java

```diff
@@ -0,0 +1,32 @@
+package edu.kit.ipd.crowdcontrol.objectservice;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.CharStreams;
+import edu.kit.ipd.crowdcontrol.objectservice.Main;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/**
+ * Utilsclass for common operations
+ *
+ * @author Marcel Hollerbach
+ */
+public class Utils {
+    /**
+     * Loads a file located relative to the main class
+     * @param file the filename to load
+     * @return the content of the file
+     */
+    public static String loadFile(String file) {
+        try {
+            return CharStreams
+                    .toString(new InputStreamReader(
+                            Main.class.getResourceAsStream(file)
+                            , Charsets.UTF_8));
+        } catch (IOException e) {
+            e.printStackTrace();
+            return "";
```

Comment:
Is this really wanted? I think it would be better if you throw an exception, if the patch is invalid, the file corrupt etc.